launcher.js

```diff
@@ -0,0 +1,45 @@
+// A "launcher" to start the (Node version of the) router from the command line.
+"use strict";
+var fluid = require("infusion");
+var flock = fluid.registerNamespace("flock");
+
+fluid.require("%gpii-launcher");
+fluid.require("%flocking-midi-router");
+
+fluid.registerNamespace("flock.midi.router.launcher");
+
+flock.midi.router.launcher.checkOptions = function (parseArgv) {
+    if (parseArgv.l || (parseArgv.inputs && parseArgv.outputs)) {
+        return true;
+    }
+    else {
+        throw "You must either specify inputs and outputs, or list ports.";
+    }
+};
+
+fluid.defaults("flock.midi.router.launcher", {
+    gradeNames: ["gpii.launcher"],
+    filterKeys: false,
+    yargsOptions: {
+        describe: {
+            "listPorts":  "List the available MIDI inputs and outputs.",
+            "inputs":     "A space-delimited array of inputs.",
+            "outputs":    "A space-delimited array of outputs.",
+            "setLogging": "The Fluid log level to use (false by default)."
+        },
+        defaults: {
+            optionsFile: "%flocking-midi-router/configs/base.json"
+        },
+        array: ["inputs", "outputs"], // TODO: Confirm this works
```

Comment:
Oh, I confirmed it by manually testing with a controller and MIDI Monitor, and didn't remove the TODO.  If you have a test input and output fixture that you use in other contexts, happy to write actual tests.